### PR TITLE
HW-5 for review

### PR DESCRIPTION
**Pull request by NatalliaSem**

### CODE REVIEW

File: Home_task_5.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+     def __init__(self, flag):
+         self.flag = flag
+
+     def choose_message_type(self):
+         if self.flag == '1':
+             self.news = News(input('Please enter news text\n'), input('Please enter location\n'))
+             news_mess = self.news
+             news_mess.news_message()
+         elif self.flag == '2':
+             self.advng = Advertising(input('Please enter advertisement text\n'), input('Please, write number of days for your add to last: \n'))
```
Comment (Verantina) [inline Home_task_5.py:83]: Read the task attentively, you need to enter the date and calculate the number of days.

File: Home_task_5.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+         self.astr_message = astr_message
+         self.zodiac_sign = zodiac_sign
+
+    def astrology_prediction(self):
+         horoscope_for_today = datetime.now().strftime("%d-%m-%Y")
+         prediction_number = random.randint(0, 10)
+         message = f'Astrology-----------------------\nHappy horoscope for today: {horoscope_for_today}\n' \
+                   f'Zodiac sign: {self.zodiac_sign}\n{self.astr_message}\n' \
+                   f'Probability of horoscope prediction from 1 to 10: {prediction_number}\n' \
+                   f'----------------------------------\n\n'
```
Comment (Verantina) [inline Home_task_5.py:52]: You create the same string in each class. Lets have this string creation as separate function in separate class and just call it if it's necessary.

File: Home_task_5.py (hunk 1 above)
Comment (Verantina) [inline Home_task_5.py:83]: What will be if I put text instead of numbers in the number of days?

Comment (NatalliaSem) [reply Home_task_5.py]: fixed

File: Home_task_5.py (hunk 2 above)
Comment (NatalliaSem) [reply Home_task_5.py]: fixed

File: Home_task_5.py (hunk 1 above)
Comment (Verantina007) [reply Home_task_5.py]: This is still not fixed, you don't have any exceptions in case of wrong format

**Comment by Verantina007** [comment]
Not all was fixed, no exceptions was added.

File: Home_task_5.py (hunk 1 above)
Comment (Verantina) [reply Home_task_5.py]: This was not fixed, your code will fail in case of wrong format.

**Comment by Verantina** [comment]
Not all was fixed, you don't have any exceptions in case of wrong format

**Pull request merged**